universal-login-sdk/lib/api/FutureWalletFactory.ts

```diff
@@ -99,7 +79,10 @@ export class FutureWalletFactory extends MineableFactory {
 
       const deployment: Deployment = {
         waitForTransactionHash: this.createWaitForTransactionHash(deploymentHash),
-        waitToBeSuccess: this.createWaitToBeSuccess(deploymentHash, deployedWallet),
+        waitToBeSuccess: async () => {
+          await this.createWaitToBeSuccess(deploymentHash)();
+          return deployedWallet;
```

Comment:
Do we want here to return deployedWallet or status?

Reply:
As I remember waitToBeSuccess should return deployedWallet and optionally we maybe will add `waitToBeMined` which will return status

Reply:
Ok yes, my mistake